apps/editing-toolkit/webpack.config.js

```diff
@@ -63,7 +63,13 @@ function getWebpackConfig( env = { source: '' }, argv = {} ) {
 		output: {
 			...webpackConfig.output,
 			path: outputPath,
-			filename: '[name].js', // dynamic filename
+			// Unfortunately, we can't set the name to `[name].js` for the
```

Comment:
I'm open to ideas on how to solve this. As I mentioned in the PR description, I think this is a good compromise to keep things simpler, but I don't like the fact that in development builds we'll have a misnamed asset JS file (with a `.min.js` extension when it's not really minified). 

I was thinking maybe another webpack plugin could go through all PHP files and change the `dist/*.js` references to either `<name>.js` (dev) or `<name>.min.js` prod, at build time 🤔

Reply:
Yeah, I think that would also make it confusing for devs because a bunch of changes would show up in VCS unexpectedly 😅 

This seems like a fair workaround, unless you want to implement SCRIPT_DEBUG. 

As a side note, I had worked on "unifying" the script enqueue logic a while ago, but never finished it. The idea was to have one function which does the enqueue/register/etc for any assets. Then, an individual project could use that without having to copy/paste a bunch of stuff from other plugins. That would make this sort of change a lot easier, so I wish I had continued that!

Reply:
As we discussed privatly, I will not add support for `SCRIPT_DEBUG`  and will consider this solution as good-enough for now. We can follow-up later.

> As a side note, I had worked on "unifying" the script enqueue logic a while ago, but never finished it. The idea was to have one function which does the enqueue/register/etc for any assets. Then, an individual project could use that without having to copy/paste a bunch of stuff from other plugins. That would make this sort of change a lot easier, so I wish I had continued that!

This sounds very interesting and would be great to have! Did you leave a WIP PR somewhere?

Reply:
Just found it! https://github.com/Automattic/wp-calypso/pull/50511/files#diff-9ba754f27ecf58cc3b55d0d053194696397eda7210d9031798dd19167049a66bR85

I linked to the most useful line, which is the `use_webpack_assets`. With that, I was able to delete a ton of other code in the plugin. I think I got stuck on the webpack changes. ETK works by having a different webpack build for each individual project, rather than one webpack build for the entire thing. The benefit is that everything can be parallelized pretty easily! But it did feel more complex than it needed to be. So I looked at turning the webpack build into just _one_ build outputting into a single dist folder (instead of multiple different ones), which enabled the use_webpack_assets function to work more easily for multiple projects.

I continued looking at webpack changes in #55261, but ultimately went with #56368 to clean up package.json scripts without drastically changing the approach. There were some drawbacks to having a single unified webpack build (maybe related to newspack blocks and overall performance??), and having that unified build was a blocker for moving forward with the PHP changes.